### DISCUSSION TITLE
Moodist V2 release?

### DISCUSSION THREAD

**Discussion by Nelinski** [remvze/moodist#45] [General]
Hi,

I've seen mention of a V2 release. Curious what features you're looking to add and if there is a rough release date?

**Comment by remvze** [comment]
Hello there,
I apologize for the delay in responding. Here's what you can expect in V2:
- More sounds
- A new brand design
- A built-in countdown timer
- A to-do checklist
- Small improvements to the remaining tools
- A binaural beat generator
- An isochronic tone generator
- Global volume control
- PWA support (this feature is tentative)
- Various small refinements to the design and functionality
- Numerous bug fixes

Regarding the release date, I don't have a specific timeline yet, as I'm currently busy with work and other projects.
Is there a particular feature you’d like to see added? I’d love to hear your feedback!

**Reply by Nelinski** [reply]
Thanks for getting back, sounds good! Main ones for me are:
- PWA (no worries if not)
- Ability to set different sounds to play at different intervals
